app/exporters/odt/cv.rb

```diff
@@ -11,9 +11,10 @@ def initialize(person, params)
 
     # rubocop:disable Metrics/MethodLength
     def export
-      template_name = anon? ? 'cv_template_anon.odt' : 'cv_template.odt'
+      template_name = "cv_template#{'_de' if location.country == 'DE'}#{'_anon' if anon?}.odt"
```

Comment:
Maybe just create a variable or method for the suffix. This looks kinda all over the place